src/pages/gen1/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -739,6 +739,74 @@ function App() {
 </Block>
 </BlockSwitcher>
 
+### Redirect URLs
```

Comment:
I think this section is redundant. By examining lines `744` and `746`, I interpret that the change we are currently implementing is the inclusion of multiple redirects, which was actually a feature offered by v6 upon its release.

Reply:
On RN page, we don't have any info on redirects so I thought this will set up some context before diving into multiple redirects.